src/components/LowSeverityTrackersWarning.vue

```diff
@@ -0,0 +1,88 @@
+<script setup lang="ts">
+import { watch } from 'vue';
+
+import { useModal } from '@/composables/useModal';
+
+import Modal from '@/widgets/Modal/Modal.vue';
+
+const props = defineProps<{
+  showModal: boolean;
+}>();
+
+const emit = defineEmits<{
+  confirmOperation: [];
+  hideModal: [];
+}>();
+
+const {
+  closeModal,
+  isModalOpen: isLowSeverityWarning,
+  openModal,
+} = useModal();
+
+watch(() => props.showModal, () => {
+  if (props.showModal) {
+    openModal();
+    return;
+  }
+  isLowSeverityWarning.value = false;
+});
+
+function cancelOperation() {
+  closeModal();
+  emit('hideModal');
+}
+
+function confirmOperation() {
+  closeModal();
+  emit('confirmOperation');
+}
+</script>
+
+<template>
+  <Modal :show="isLowSeverityWarning" style="max-width: 50%; margin-top: 6ch;" @close="cancelOperation()">
+    <template #body>
+      <div class="alert alert-warning p-2 mb-0">
+        <h1 class="modal-title fs-5 mb-2">
+          <div class="badge text-bg-warning mx-1 fs-6">
+            <i class="bi bi-exclamation-triangle-fill" />
+            Warning
+          </div>
+          Filing Low Severity Trackers
+        </h1>
+        <button
+          type="button"
+          class="btn-close"
+          aria-label="Close"
+          @click="cancelOperation()"
+        />
```

Comment:
Should be this inside of the `#header` slot instead of the body?

Reply:
To give all the modal a warn alert style it was simpler to me to wrap everything under the body with the alert-warning div than having individual divs in the header, body and footer repeating that class.

My aim for conditionally render the header is to be able to use only the body in some cases were it is intended to don't reuse that template.